pages/operator-address.js

```diff
@@ -20,7 +20,10 @@ const OperatorAddress = (props) => (
       validatorErrors={props.validatorErrors}
       onHandleErrorClick={OnHandleErrorClick}
     />
```

Comment:
Same establishment address

Reply:
did the same as I did with the establishment address but since here it calls the addresshelp component, I passed the id to it and did the implementation on the actual component